shared/common-adapters/header-hoc/index.native.tsx

```diff
@@ -124,27 +124,26 @@ export const LeftAction = ({
   customIconColor,
 }: LeftActionProps): React.ReactElement => (
   <Box style={Styles.collapseStyles([styles.leftAction, hasTextTitle && styles.grow])}>
-    {onLeftAction &&
-      (leftAction === 'cancel' ? (
-        <Text type="BodyBigLink" style={styles.action} onClick={onLeftAction}>
-          {leftActionText || customCancelText || 'Cancel'}
-        </Text>
-      ) : (
-        <BackButton
-          badgeNumber={badgeNumber}
-          hideBackLabel={hideBackLabel}
-          iconColor={
-            customIconColor ||
-            (disabled
-              ? Styles.globalColors.black_10
-              : theme === 'dark'
-              ? Styles.globalColors.white
-              : Styles.globalColors.black_50)
-          }
-          style={styles.action}
-          onClick={onLeftAction}
-        />
-      ))}
+    {onLeftAction && leftAction === 'cancel' ? (
```

Comment:
There's a back button on the root of the people tab now on mobile, maybe this change?

Reply:
good find, changed to be more backwards compatible

Reply:
lgtm :+1: